### DISCUSSION TITLE
Ability to select drivers in Live Timing window

### DISCUSSION THREAD

**Discussion by kaitbrecosky** [f1multiviewer/issue-tracker#387] [Feature requests]
Currently, the Live Timing window uses a bunch of CPU power to display and update all 20 drivers and their associated data live during a race. My feature request is to have the ability to select which drivers you would like to load live data for, to reduce CPU usage and associated stuttering while it is syncing.

By allowing users to select which drivers they want this data for, this can improve stability of program and streams. This would directly benefit users that have older machines or lower spec computers that they would like to utilize Multiviewer to watch F1, but can't use certain features because of the amount of processing power that is required to have it run smoothly or not impact the quality of live stream feeds.

To implement this in the current setup, there could be either a +/- sign to add and remove drivers from the list of drivers that you want live timing data for, or there could be a toggle switch that will start/stop live timing data for a driver.

**Comment by jeppeolesen** [comment]
Marking this as a duplicate for #19
